www/src/pages/docs/50_StoreMapping.md

```diff
@@ -311,8 +308,10 @@ val person = Person(Address("Lerchenweg"))
 streetOfPerson.get(person) // -> "Lerchenweg"
 streetOfPerson.set("Rosenstraße") // Person(address = Address("Rosenstraße"))
 ```
+This works, but the syntax is quite cumbersome - especially for deeper nested models.
```

Comment:
This syntax is not meant! The `+` Operator is cumbersome for deeper nested models ;-)
So the old appearence is valid.